runtime/picasso/src/governance.rs

```diff
@@ -0,0 +1,163 @@
+//! Runtime setup for the governance and democracy only.
+
+use super::*;
+use common::governance::native::*;
+use frame_support::traits::LockIdentifier;
+
+pub type NativeDemocracy = democracy::Instance1;
+
+pub type NativeCouncilMembership = membership::Instance1;
+pub type NativeTechnicalMembership = membership::Instance2;
+
+parameter_types! {
+	pub const CouncilMotionDuration: BlockNumber = 7 * DAYS;
+	pub const CouncilMaxProposals: u32 = 100;
+	pub const CouncilMaxMembers: u32 = 100;
+}
+
+impl membership::Config<NativeCouncilMembership> for Runtime {
+	type Event = Event;
+	type AddOrigin = EnsureRootOrHalfNativeCouncil;
+	type RemoveOrigin = EnsureRootOrHalfNativeCouncil;
+	type SwapOrigin = EnsureRootOrHalfNativeCouncil;
+	type ResetOrigin = EnsureRootOrHalfNativeCouncil;
+	type PrimeOrigin = EnsureRootOrHalfNativeCouncil;
+	type MembershipInitialized = Council;
+	type MembershipChanged = Council;
+	type MaxMembers = CouncilMaxMembers;
+	type WeightInfo = weights::membership::WeightInfo<Runtime>;
+}
+
+impl collective::Config<NativeCouncilCollective> for Runtime {
+	type Origin = Origin;
+	type Proposal = Call;
+	type Event = Event;
+	type MotionDuration = CouncilMotionDuration;
+	type MaxProposals = CouncilMaxProposals;
+	type MaxMembers = CouncilMaxMembers;
+	type DefaultVote = collective::PrimeDefaultVote;
+	type WeightInfo = weights::collective::WeightInfo<Runtime>;
+}
+
+impl membership::Config<NativeTechnicalMembership> for Runtime {
+	type Event = Event;
+	type AddOrigin = EnsureRootOrHalfNativeTechnical;
+	type RemoveOrigin = EnsureRootOrHalfNativeTechnical;
+	type SwapOrigin = EnsureRootOrHalfNativeTechnical;
+	type ResetOrigin = EnsureRootOrHalfNativeTechnical;
+	type PrimeOrigin = EnsureRootOrHalfNativeTechnical;
+	type MembershipInitialized = TechnicalCollective;
+	type MembershipChanged = TechnicalCollective;
+	type MaxMembers = CouncilMaxMembers;
+	type WeightInfo = weights::membership::WeightInfo<Runtime>;
+}
+
+impl collective::Config<NativeTechnicalMembership> for Runtime {
+	type Origin = Origin;
+	type Proposal = Call;
+	type Event = Event;
+	type MotionDuration = CouncilMotionDuration;
+	type MaxProposals = CouncilMaxProposals;
+	type MaxMembers = CouncilMaxMembers;
+	type DefaultVote = collective::PrimeDefaultVote;
+	type WeightInfo = weights::collective::WeightInfo<Runtime>;
+}
+
+// NOTE: this is for testing runtime to fast track
+parameter_types! {
+	// TODO: replace at some point later with these
+	// 	pub const LaunchPeriod: BlockNumber = 5 * DAYS;
+	// 	pub const FastTrackVotingPeriod: BlockNumber = 3 * HOURS;
+	// 	pub const EnactmentPeriod: BlockNumber = 2 * DAYS;
```

Comment:
Are we fixing that in this PR? Prefer not to ever have bad values in our configs.

Reply:
Yes. Will leave `FastTrackVotingPeriod` only to be smaller. LaunchPeriod and EnactmentPeriod are as I readd can be overridden by all yes council simple super majority refernad creation (no stake weighted) and make enactment(dispatch) as soon as needed.